### PR TITLE
Feature/dateimplementation

### PR DESCRIPTION
**Pull request by swapnatootikoora1**
Feature/dateimplementation

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/api/service/NiccService.scala [hunk 1]
```diff
@@ -28,17 +28,15 @@ import scala.concurrent.duration._
...
+  val host: String  = TestConfiguration.url("nicc")
+  val token: String = new AuthService().postLogin.headers
     .get("Authorization")
     .flatMap(_.headOption)
     .getOrElse(fail("Couldn't retrieve Auth Token"))
 
   def makeRequest(request: Request, timeoutDuration: Int = 10): StandaloneWSRequest#Self#Response = {
 
-    val url: String = s"$host/contribution-and-credits/"
+    val url: String    = s"$host/contribution-and-credits/"
```
Comment (MartinTGeorgiev) [inline src/test/scala/uk/gov/hmrc/api/service/NiccService.scala:39]: Remove this extra space.

File: src/test/scala/uk/gov/hmrc/api/service/NiccService.scala [hunk 2]
```diff
@@ -53,7 +51,7 @@ class NiccService extends HttpClient {
 
   def makeRequestWithBearerToken(request: Request, bearerToken: String): StandaloneWSRequest#Self#Response = {
 
-    val url: String = s"$host/contribution-and-credits/"
+    val url: String    = s"$host/contribution-and-credits/"
```
Comment (MartinTGeorgiev) [inline src/test/scala/uk/gov/hmrc/api/service/NiccService.scala:54]: Remove this extra space.

**Review by MartinTGeorgiev — CHANGES_REQUESTED** [review]
Fix comments mentioned

**Review by MartinTGeorgiev — APPROVED** [review]

File: src/test/scala/uk/gov/hmrc/api/service/NiccService.scala (hunk 1 above)
Comment (swapnatootikoora1) [reply src/test/scala/uk/gov/hmrc/api/service/NiccService.scala]: removed

File: src/test/scala/uk/gov/hmrc/api/service/NiccService.scala (hunk 2 above)
Comment (swapnatootikoora1) [reply src/test/scala/uk/gov/hmrc/api/service/NiccService.scala]: removed

**Pull request merged**